packages/site/src/main.css

```diff
@@ -40,51 +40,55 @@ a:visited {
   color: var(--color-text--secondary);
 }
 
-custom-elements {
+custom-elements,
+section[role="tabpanel"] {
   & table {
-    font-size: 16px;
-    margin: 0;
+    font-size: var(--typography--fontSize-large);
+    min-width: 66%;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
     -webkit-overflow-scrolling: touch;
-    margin: 16px 0;
-    margin-bottom: 16px;
+    margin: var(--space-large) auto;
     font-size: 14px;
     line-height: 24px;
     padding: 0;
     border-collapse: collapse;
+    box-shadow: var(--shadow-low);
+    border-radius: var(--radius-base);
```

Comment:
When a table has `border-collapse` applied, `border-radius` stops working, so the box-shadow gives us the visual outline without using a border! (You can also see this in the Atlantis `<Table>` component, thanks @eddysims for showing me this all those years ago)